MapboxNavigation/ExitView.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  ExitView.swift
+//  MapboxNavigation
+//
+//  Created by Jerrad Thramer on 3/23/18.
+//  Copyright © 2018 Mapbox. All rights reserved.
+//
+
+import UIKit
+
+enum ExitSide: String{
+    case left, right, other
+    
+    var exitImage: UIImage {
+        return self == .left ? ExitView.leftExitImage : ExitView.rightExitImage
+    }
+}
+
+class ExitView: UIView {
+    static let leftExitImage = UIImage(named: "exit-left", in: .mapboxNavigation, compatibleWith: nil)!.withRenderingMode(.alwaysTemplate)
+    static let rightExitImage = UIImage(named: "exit-right", in: .mapboxNavigation, compatibleWith: nil)!.withRenderingMode(.alwaysTemplate)
+    
+    var side: ExitSide = .right {
+        didSet {
+            populateExitImage()
+            rebuildConstraints()
+        }
+    }
+    
+    lazy var imageView: UIImageView = {
+        let view = UIImageView(image: self.side.exitImage)
+        view.tintColor = .black
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.contentMode = .scaleAspectFit
+        return view
+    }()
+    
+    lazy var exitNumberLabel: UILabel = {
+        let label: UILabel = .forAutoLayout()
+        label.text = exitText
+        label.textColor = .black
+        print("system font size! \(UIFont.systemFontSize)")
+        label.font = UIFont.boldSystemFont(ofSize: 20)
+        return label
+    }()
+
+    var exitText: String? {
+        didSet {
+            exitNumberLabel.text = exitText
+            invalidateIntrinsicContentSize()
+        }
+    }
+    var pointSize: CGFloat? {
```

Comment:
Can this be non-optional since it's set during init?

Reply:
It could default to `PrimaryLabel.appearance().normalFont`